ros/src/tl_detector/tl_detector.py

```diff
@@ -33,7 +33,11 @@ def __init__(self):
 
         self.bridge = CvBridge()
         self.light_classifier = TLClassifier(self.config['tl_model_path'], min_detect_score_thresh=self.config['min_detect_tl_score_thresh'])
-        #self.listener = tf.TransformListener()
+        self.tl_start_detection_distance = float(self.config['tl_start_detection_distance']) if self.config['tl_detection_from_topic'] else 100
```

Comment:
I think it will fail here if the option is not present in the config. try the following:
`self.config.get('tl_start_detection_distance', 100)` instead

Reply:
True